eng/build.ps1

```diff
@@ -14,6 +14,12 @@
 # it's fine to call `build.ps1 -build -testDesktop` followed by repeated calls to
 # `.\build.ps1 -testDesktop`.
 
+if ($PSVersionTable.PSVersion.Major -lt "5") {
+   $version = $PSVersionTable.PSVersion
+   Write-Host "PowerShell version must be 5 or greater (version $version detected)"
+               exit 1
```

Comment:
To much indenting. `exit 1` should be indented the same as `Write-Host`.

Reply:
Done